docs_sources/erp_fhirversion_change_20250115-source.adoc

```diff
@@ -0,0 +1,117 @@
+= E-Rezept FHIR Veränderungen für 20250115 image:gematik_logo.png[width=150, float="right"]
+include::./config-source.adoc[]
+
+Ab 15.01.2025 ist das gematik Workflow Package 1.4.0 gültig. Da sowohl DiGA als auch eML Teil des Packages sind, müssen einige Veränderungen vorgenommen werden. Diese Seite bietet eine Zusammenfassung der Veränderungen für die verschiedenen Hersteller.
+
+toc::[]
+
+== Versionsübergabegänge
+Ab dem 01.11.2024 kann der E-Rezept-Fachdienst mehrere aktive Profile haben. Das bedeutet, dass Übergangszeiten mit mehr als zwei aktiven Profilen möglich sind. Unten sind sowohl ein Diagramm als auch eine Tabelle zu sehen, die zeigen, wann welches Profil aktiv ist und bis wann.
+
+image:puml_workflow_profile_version_timeline.png[]
+
+|===
+h|Version  h|Gültig ab  h|Gültig bis
+|Profile 1.2| - | 15.07.2025
+|Profile 1.3| 01.11.2024 | 15.07.2025
+|Profile 1.4| 15.01.2025 | - 
+|Profile 1.5 (Vorläufig)| 15.07.2025 | -
+|===
+
+NOTE: Zwischen dem 15.04.2025 und dem 15.07.2025 sind Profile 1.2 und 1.3 nur für die FdVs verfügbar.
+NOTE: Ab dem 15.07.2025 ist Profil 1.4 nur für die FdVs verfügbar.
+
+== Änderungen
+=== Allgemein
+Betrifft: image:{PVS}[] image:{AVS}[] image:{FdV}[]
+
+* Meta.Profile hochziehen und auf 1.4 ändern. Das betrifft alle Ressourcen.
```

Comment:
Könnte das etwas sachlicher und technischer beschrieben werden? "Hochziehen" klingt etwas zu umgangsprachlich.
Besser sowas wie "Anpassen von meta.profile von.. auf..."

Reply:
Jetzt ist der Punkt: 
"Anpassen von meta.profile von 1.3 auf 1.4. Das betrifft alle Ressourcen."

Reply:
Ja eher "Anpassen der Versionsangabe in meta.profile von '<profil-cannonical>|1.3' zu '<profil-cannonical>|1.4'
Ist etwas genauer